.cloudgov/manifest.yml

```diff
@@ -6,6 +6,7 @@ applications:
     instances: 0
     docker:
       image: ((image))
+      username: AKIAWNYQRJFGHBAWM5MI
```

Comment:
should we put this in a circleci env variable?

Reply:
we can, we don't have to. Per prior, unrelated convos in Slack, the account key is NOT secret

Reply:
ok ... good to know ... ideally, i'd prefer it in circleci env vars

Reply:
It's also part of the ECR repository URL so I'd have to string concat with the env var in multiple places as well...

Reply:
Another reason for leaving it is I can still use the same deploy script since I don't need to pass it as an extra CL parameter. That said, this won't be an issue when both staging and prod are updated, let's revisit.